modules/virt-inferfromvolume-labels.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-specifying-instance-preference.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="inferfromvolume-labels_{context}"]
+= Setting the `inferFromVolume` labels
+
+Use the following labels on your PVC, data source, or data volume to instruct the inference mechanism which instance type, preference, or both to use when trying to boot from a volume.
+
+* A cluster-wide instance type: `instancetype.kubevirt.io/default-instancetype`
+* A namespaced instance type: `instancetype.kubevirt.io/default-instancetype-kind`
+** Optional. Otherwise, defaults to `VirtualMachineClusterInstancetype`.
+* A cluster-wide preference: `instancetype.kubevirt.io/default-preference`
+* A namespaced preference: `instancetype.kubevirt.io/default-preference-kind`
+** Optional. Otherwise, defaults to `VirtualMachineClusterPreference`.
+
+.Prerequisites
+
+* You must have an instance type, preference, or both on the cluster.
+
+.Procedure
+
+. To apply a label to a data source, use the `kubectl/oc label`. The following example applies a label to a cluster-wide instance type:
```

Comment:
```suggestion
. To apply a label to a data source, use the `oc label`. The following example applies a label to a cluster-wide instance type:
```

If this is purely OpenShift specific then `oc` suffices. I was not sure which tool to name initially.